test/unit/rules/require-valid-alt-text-test.js

```diff
@@ -11,13 +11,16 @@ generateRuleTests({
     '<img alt="blah {{derp}}">',
     '<img aria-hidden="true">',
     '<img hidden>',
+    '<img alt="" src="zoey.jpg">',
     '<img alt="" role="none" src="zoey.jpg">',
     '<img alt="" role="presentation" src="zoey.jpg">',
     '<img alt="a stylized graphic of a female hamster" src="zoey.jpg">',
+    '<img alt="a stylized graphic of a female hamster" role="presentation" src="zoey.jpg">',
```

Comment:
`role="presentation"` and `role="none"` are not valid when `alt` is a non-empty string